WalletWasabi/Blockchain/Keys/KeyManager.cs

```diff
@@ -128,6 +129,10 @@ public KeyManager(BitcoinEncryptedSecretNoEC encryptedSecret, byte[] chainCode,
 		[JsonProperty(Order = 11, PropertyName = "PreferPsbtWorkflow")]
 		public bool PreferPsbtWorkflow { get; set; }
 
+		[DefaultValue(true)]
```

Comment:
The default value must be `false`.

Reasoning:
This is important because of backward compatibility. An old wallet that was created in 1.0 will not know about this property, so the value will be set to `true`. Wasabi would start CJing automatically and since it has a cost the user would see they lost some funds. That would cause a serious problem for us.

Reply:
Done